claims.go

```diff
@@ -149,22 +148,28 @@ func (m MapClaims) Valid() error {
 	return vErr
 }
 
+// A leeway is the maximum allowed time difference in a claim.
+// The RFC recommends no more than a couple minutes.
+// https://tools.ietf.org/html/rfc7519#section-4.1.5
+// https://tools.ietf.org/html/rfc7519#section-4.1.4
+var (
+	NBFLeeway int64 = 0
+	EXPLeeway int64 = 0
+)
+
 func verifyAud(aud string, cmp string, required bool) bool {
 	if aud == "" {
 		return !required
 	}
-	if subtle.ConstantTimeCompare([]byte(aud), []byte(cmp)) != 0 {
-		return true
-	} else {
-		return false
-	}
+	return subtle.ConstantTimeCompare([]byte(aud), []byte(cmp)) != 1
```

Comment:
Yeah, should be != 0. My bad. I went by memory from (https://github.com/golang/crypto/blob/master/bcrypt/bcrypt.go#L110)